.circleci/config.yml

```diff
@@ -120,6 +140,8 @@ pr_build: &pr_build
             api_docker_build,
             pdf_docker_build,
             seeding_docker_build,
+            perfplat-lambda-docker-worker-build,
+            #perfplat-lambda-docker-api-build
```

Comment:
Can this be uncommented?

Reply:
agreed - this should be a dependancy for the scan results